Docker/validate_sc1.py

```diff
@@ -73,14 +73,11 @@ def main(submission, entity_type, goldstandard, results):
 
     invalid_reasons = set()
 
-    # Validation 1: submission is a file, not a Synapse project.
     if submission is None:
         invalid_reasons = {
             f"Expected FileEntity type but found {entity_type}"}
     else:
         with open(submission) as pred:
-
-            # Validation 2: only two columns: dreamID, nw
             error = check_header(pred.readline())
```

Comment:
Comments are always welcome.  No need to add back in though.

Reply:
haha it didn't look nice with the levels of indents I had...